integration/test_graphql.py

```diff
@@ -108,6 +108,44 @@ def test_get_data(client):
     assert a_found and d_found and len(objects) == 2
 
 
+def test_get_data_with_properties_default(client, people_schema):
+    """Test GraphQL's Get clause with properties default (None) and no additional properties specified."""
+    client.schema.create(people_schema)
+
+    person_id = uuid.uuid4()
+    group_id = uuid.uuid4()
+    client.data_object.create(
+        {
+            "name": "John",
+        },
+        "Person",
+        person_id,
+    )
+
+    client.data_object.create(
+        {
+            "name": "Group-Weaviate",
+        },
+        "Group",
+        group_id,
+    )
+
+    client.data_object.reference.add(
+        group_id,
+        "member",
+        person_id,
+        from_class_name="Group",
+        to_class_name="Person",
+    )
+
+    result_without_additional_props = client.query.get("Group").with_limit(1).do()
+    result_with_additional_props = client.query.get("Group").with_limit(1).with_additional(["id"]).do()
+    objects_without_additional_props = get_objects_from_result(result_without_additional_props)
+    objects_with_additional_props = get_objects_from_result(result_with_additional_props)
+
+    assert "name" in objects_without_additional_props[0]
+    assert "id" in objects_with_additional_props[0]["_additional"]
```

Comment:
This check also needs to include that "name" is NOT part of the answer

Reply:
Added